o3po/includes/class-o3po-settings.php

```diff
@@ -117,13 +117,14 @@ class O3PO_Settings extends O3PO_Singleton {
         'arxiv_url_source_prefix' => 'https://arxiv.org/e-print/',
         'arxiv_url_trackback_prefix' => 'http://arxiv.org/trackback/',
         'doi_url_prefix' => 'https://doi.org/',
-        'scirate_url_abs_prefix' => 'https://scirate.com/arxiv/',
+        'scirate_url_abs_prefix' => '',
```

Comment:
Please leave the default set and instead add a sentence to the render function corresponding to this settings field that explains that by making the field empty the link to scirate can be suppressed.

Reply:
I tried this but it kept applying the default when the field set to the empty string.

Reply:
Then there is an error with how the defaults are set in the settings class. We should identify and fix this error and keep the default.

Reply:
> We should identify and fix this error and keep the default.

Sure, we can do that!